subsys/bluetooth/host/classic/avdtp.c

```diff
@@ -340,602 +339,594 @@ static struct bt_avdtp_sep *avdtp_get_sep(uint8_t stream_endpoint_id)
+{
+	struct bt_avdtp_req *req = session->req;
+
+	if (req == NULL) {
+		return;
+	}
+	k_work_cancel_delayable(&session->timeout_work);
+	if (msg_type == BT_AVDTP_ACCEPT) {
+		bt_avdtp_set_state_lock(CTRL_REQ(req)->sep, AVDTP_OPEN);
+	} else if (msg_type == BT_AVDTP_REJECT) {
+		avdtp_handle_reject(buf, req);
+	}
+
+	if (req->status == BT_AVDTP_SUCCESS) {
 		avdtp_set_status(req, buf, msg_type);
-		bt_avdtp_clear_req(session);
-		if (req->func != NULL) {
-			req->func(req);
-		}
+	}
+	bt_avdtp_clear_req(session);
+	if (req->func != NULL) {
+		req->func(req);
 	}
 }
 
-static void avdtp_abort_handler(struct bt_avdtp *session, struct net_buf *buf, uint8_t msg_type,
-				uint8_t tid)
+static void avdtp_abort_cmd(struct bt_avdtp *session, struct net_buf *buf, uint8_t tid)
 {
-	if (msg_type == BT_AVDTP_CMD) {
-		int err = 0;
-		int ret;
-		struct bt_avdtp_sep *sep;
-		struct net_buf *rsp_buf;
-		uint8_t error_code = 0;
+	int err = 0;
+	int ret;
+	struct bt_avdtp_sep *sep;
+	struct net_buf *rsp_buf;
+	uint8_t error_code = 0;
 
-		sep = avdtp_get_cmd_sep(buf);
-		avdtp_sep_lock(sep);
-		if ((sep == NULL) || (session->ops->abort_ind == NULL)) {
-			err = -ENOTSUP;
-		} else {
-			/* all current sep state is OK for abort operation */
-			err = session->ops->abort_ind(session, sep, &error_code);
-		}
+	sep = avdtp_get_cmd_sep(buf, &error_code);
+	avdtp_sep_lock(sep);
+	if ((sep == NULL) || (session->ops->abort_ind == NULL)) {
+		err = -ENOTSUP;
+	} else {
+		/* all current sep state is OK for abort operation */
+		err = session->ops->abort_ind(session, sep, &error_code);
+	}
 
-		rsp_buf = avdtp_create_reply_pdu(err ? BT_AVDTP_REJECT : BT_AVDTP_ACCEPT,
-						 BT_AVDTP_PACKET_TYPE_SINGLE, BT_AVDTP_ABORT, tid);
-		if (!rsp_buf) {
-			avdtp_sep_unlock(sep);
-			return;
-		}
+	rsp_buf = avdtp_create_reply_pdu(err ? BT_AVDTP_REJECT : BT_AVDTP_ACCEPT,
+					 BT_AVDTP_PACKET_TYPE_SINGLE, BT_AVDTP_ABORT, tid);
+	if (!rsp_buf) {
+		avdtp_sep_unlock(sep);
+		return;
+	}
 
-		if (err) {
-			if (error_code == 0) {
-				error_code = BT_AVDTP_BAD_ACP_SEID;
-			}
-			LOG_DBG("abort err code:%d", error_code);
-			net_buf_add_u8(rsp_buf, error_code);
+	if (err) {
+		if (error_code == 0) {
+			error_code = BT_AVDTP_BAD_ACP_SEID;
 		}
+		LOG_DBG("abort err code:%d", error_code);
+		net_buf_add_u8(rsp_buf, error_code);
+	}
 
-		ret = bt_l2cap_chan_send(&session->br_chan.chan, rsp_buf);
-		if (ret) {
-			net_buf_unref(rsp_buf);
-			LOG_ERR("Error:L2CAP send fail - result = %d", ret);
-		}
-		if (!err && !ret) {
-			if ((sep->state & (AVDTP_OPEN | AVDTP_STREAMING)) &&
-			    (sep->chan.state == BT_L2CAP_CONNECTED)) {
-				bt_avdtp_set_state(sep, AVDTP_ABORTING);
-			} else {
-				bt_avdtp_set_state(sep, AVDTP_IDLE);
-			}
+	ret = bt_l2cap_chan_send(&session->br_chan.chan, rsp_buf);
+	if (ret) {
+		net_buf_unref(rsp_buf);
+		LOG_ERR("Error:L2CAP send fail - result = %d", ret);
+	}
+	if (!err && !ret) {
```

Comment:
Same thing here. There's a general convention in Zephyr to use either `err` or `ret` for `int` style generic error variable. The convention in Bluetooth code has been to use `err`. However, it gets really confusing when you have both - please don't do that. Note: I can see that this problem exists even before this PR, so fixing it doesn't necessarily have to be done in this commit/PR as long as it get fixed ASAP.

Reply:
Hi @jhedberg  I add one commit to address all the blank line issue, please help to review. One exception that I do is: if the code `if (xx) {`'s condition use the previous simple code line's result, I put the previous simple codes lines near with the `if (xx) {`. like: https://github.com/zephyrproject-rtos/zephyr/pull/83375/files#diff-aebc3083e23e5c1f24f55bc6bf88e71ffa3ddc8dbae92db402bd4fa8402aa461L585-L587
This is my understand. please help to check whether it is right.

For `err` and `ret`, I plan to change `err` as `cmd_err` because it is result of processing the cmd, change `ret` as `l2cap_ret` because it is the l2cap send api return value in next pr after this pr is merged.